src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkTask.java

```diff
@@ -77,6 +79,7 @@ public void start(Map<String, String> props, JestClient client) {
       int maxRetry = config.getInt(ElasticsearchSinkConnectorConfig.MAX_RETRIES_CONFIG);
       int connTimeout = config.getInt(ElasticsearchSinkConnectorConfig.CONNECTION_TIMEOUT_MS_CONFIG);
       int readTimeout = config.getInt(ElasticsearchSinkConnectorConfig.READ_TIMEOUT_MS_CONFIG);
+      BehaviorOnNullValues behaviorOnNullValues = BehaviorOnNullValues.forValue(config.getString(ElasticsearchSinkConnectorConfig.BEHAVIOR_ON_NULL_VALUES_CONFIG));
```

Comment:
People get sloppy and sometimes include leading or trailing whitespace. Might want to `trim()` the string in the `BehaviorOnNullValues.forValue(String)` method before uppercasing it.

Reply:
Looks like the `AbstractConfg` class's `getString()` method already strips leading and trailing whitespace; is it still worth adapting for the cases where someone explicitly invokes `BehaviorOnNullValues.forValue()` themselves?

Reply:
Ah, good catch. No need to change anything then.

Reply:
@C0urante `getString` does not trim the value. It returns what's in the map of parsed values. 

The string value is parsed when it's added to the map as part of the overall parsing of the original properties.

Reply:
@kkonstantine right, point still stands though--what `getString()` returns is the trimmed version of what the `AbstractConfig` was given to begin with.